typesense-java/src/main/java/org/typesense/model/Api.java

```diff
@@ -11,8 +11,10 @@
 import org.slf4j.Logger;
```

Comment:
Let's call this file `ApiCall.java` to maintain consistency with other clients

Reply:
Sure!